<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anagrams</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>701</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Anagrams</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createGameButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joinGameButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Join Game</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputIPAddress">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter IP Address:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
